gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -26,10 +26,54 @@ Dump::Dump (std::ostream &stream) : stream (stream) {}
 void
 Dump::go (HIR::Crate &crate)
 {
-  // TODO: print crate inner_attrs
-  // TODO: print crate items
-  // TODO: print crate mappings
-  stream << crate.as_string ();
+  stream << "Crate"
+	 << " "
+	 << "{" << std::endl;
+  //
+
+  indent++;
+  stream << std::string (indent, indent_char);
+  stream << "inner_attrs"
+	 << ":"
+	 << " "
+	 << "[";
```

Comment:
I think this is fine for now, but we should open up an issue about factoring this in a function as you use it often :)

Reply:
Yeah will do.